include/db/object.h

```diff
@@ -33,6 +35,8 @@ class LocalObject : public QEnableSharedFromThis<LocalObject> {
   Universe *db_;
   QString name_;
   QString comment_;
+  static std::atomic<uint64_t> static_id_;
+  uint64_t id_;
```

Comment:
What do you think about also adding version field? We don't need it before we start modifying objects over the network, but once we do allow that we need a way to defend against race conditions on updates.